### ISSUE TITLE
Assignment 1 - Link to Amsterdam Demographics does not work 

### ISSUE THREAD

**Issue by Nienmaison** [TPM034A/Q2_2023#4]
Hi, 

The link that explains the demographic statistics of Amsterdam provided in the Assignment does not work. Do you have the URL? 

Kind regards, 

Daan

**Comment by Nienmaison** [comment]
I think I found it when opening the metadata.csv. Is this correct?

**Comment by spierenbu** [comment]
Yes, the metadata.csv file is the target of the link. The link works on my computer, I will investigate why it does not work on your side.
